LoopTests/Mock Stores/HKHealthStoreMock+LoopTests.swift

```diff
@@ -0,0 +1,76 @@
+//
```

Comment:
Copy-pased directly from LoopKit without any edits

Reply:
I am guessing this is being included because all the reaching into LDM for the HKHealthStore. I think it'd be nice to move the authorization stuff into DeviceDataManager, and keep it out of LDM. Then we wouldn't need to bring the mock over; HK access really should be the domain of the stores, and LDM should't worry about it at all.

Reply:
This is actually being included so the mock stores can be generated (like `MockDoseStore`, etc) for testing LDM. I think we'd need to bring this mock over regardless, unless the mocks should be located in LoopKit instead of LoopTests

Reply:
I know the HKHealthStore is referenced in the constructor of the stores, but the constructor isn't part of the protocol. Where else is the HKHealthStore used in the mocks?

Reply:
`InsulinDeliveryStore` is part of the `DoseStoreProtocol`, and `InsulinDeliveryStore` requires a `HKHealthStore`. I can mock the `InsulinDeliveryStore` if we want to pull out the `HKHealthStoreMock`; `HKHealthStoreMock` isn't being used in a meaningful way for these tests, so it's not critical that it be there, but I was also hesitant to create another mock if it wasn't necessary

Reply:
I mocked `InsulinDeliveryStore` and initialized the stores/moved the logic for authorization into the device manager